src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini

```diff
@@ -286,3 +286,8 @@ COM_TJNOTIFICATIONS_NOTIFICATION_SMS_REMAINING_EXCEEDED="characters exceeded"
 ; Notification - SMS provider template ID
 COM_TJNOTIFICATIONS_FIELD_SMS_TEMPLATE_ID_LABEL="SMS template ID"
 COM_TJNOTIFICATIONS_FIELD_SMS_TEMPLATE_ID_DESC="Set template ID which is proided by SMS provider"
+
+;Notification - Email service provider config
+COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_EMAIL_PROVIDER="Choose email service provider plugin"
+COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_EMAIL_PROVIDER_DESC="Choose email service provider plugin to be used to send an emails"
+COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_EMAIL_DEFAULT_OPTION="Joomla SMTP"
```

Comment:
@tushar33 Joomla SMTP => Joomla Mail
as SMTP is one of the Mail settings in Joomla

Reply:
done